<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Ai_Lab2_qtv2Class</class>
 <widget class="QMainWindow" name="Ai_Lab2_qtv2Class">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>931</width>
    <height>592</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Ai_Lab2_qtv2</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGraphicsView" name="graphicsView">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>20</y>
      <width>661</width>
      <height>561</height>
     </rect>
    </property>
   </widget>
   <widget class="QListWidget" name="listWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>211</width>
      <height>192</height>
     </rect>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBoxQuad">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>250</y>
      <width>101</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Quadrangulate</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonOptimize">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>280</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Optimize</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonPath">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>480</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Path</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="doubleGenerationTime">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>220</y>
      <width>62</width>
      <height>22</height>
     </rect>
    </property>
    <property name="buttonSymbols">
     <enum>QAbstractSpinBox::NoButtons</enum>
    </property>
    <property name="maximum">
     <double>999999.989999999990687</double>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>220</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Generation Time (ms)</string>
    </property>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>330</y>
      <width>141</width>
      <height>134</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QRadioButton" name="radioAStar">
       <property name="text">
        <string>A*</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="radioDijkstra">
       <property name="text">
        <string>Dijkstra</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="radioBreadth">
       <property name="text">
        <string>Breadth First</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="radioDepth">
       <property name="text">
        <string>Depth First</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="radioTurnLeft">
       <property name="text">
        <string>Turn Left</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="radioRandom">
       <property name="text">
        <string>Random</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>520</y>
      <width>160</width>
      <height>51</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QDoubleSpinBox" name="doublePathLength">
       <property name="maximumSize">
        <size>
         <width>80</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="buttonSymbols">
        <enum>QAbstractSpinBox::NoButtons</enum>
       </property>
       <property name="maximum">
        <double>999999.989999999990687</double>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Length</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QDoubleSpinBox" name="doubleTimeElapsed">
       <property name="maximumSize">
        <size>
         <width>80</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="buttonSymbols">
        <enum>QAbstractSpinBox::NoButtons</enum>
       </property>
       <property name="maximum">
        <double>999999999.990000009536743</double>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>ms</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="ai_lab2_qtv2.qrc"/>
 </resources>
 <connections/>
</ui>
